examples/multiphysics/static-bubble/static-bubble.prm

```diff
@@ -0,0 +1,154 @@
+  set time step        = 0.005
+  set output name      = static-bubble
+  set output frequency = 0
+  set output path      = ./output/
+  set subdivision       = 3
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+
+subsection multiphysics
+  set VOF = true
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type               = dealii
+  set grid type          = hyper_rectangle
+  set grid arguments     = -2.5, -2.5 : 2.5, 2.5 : true
+  set initial refinement = 7
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+
+subsection initial conditions
+  set type = nodal
+  subsection uvwp
+    set Function expression = 0; 0; 0
+  end
+  subsection VOF
+    set Function expression = if (x * x + y * y < 0.5 * 0.5 , 1, 0)
+    subsection projection step
+      set enable = true
+      set diffusion factor = 1
+    end
+  end
+end
+
+#---------------------------------------------------
+# VOF
+#---------------------------------------------------
+
+subsection VOF
+  subsection surface tension force
+    set enable                         = true
+    set surface tension coefficient    = 1
+    set phase fraction gradient filter factor = 4
+    set curvature filter factor               = 1
+    set output auxiliary fields        = true
+  end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection physical properties
+  set number of fluids = 2
+  subsection fluid 1
+    set density             = 10
+    set kinematic viscosity = 0.1
+  end
+  subsection fluid 0
+    set density             = 10
+    set kinematic viscosity = 0.1
+  end
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+
+subsection boundary conditions
+  set number = 4
+  subsection bc 0
+    set id   = 0
+    set type = noslip
+  end
+  subsection bc 1
+    set id   = 1
+    set type = noslip
+  end
+  subsection bc 2
+    set id   = 2
+    set type = noslip
+  end
+  subsection bc 3
+    set id   = 3
+    set type = noslip
+  end
+end
+
+#---------------------------------------------------
+# Analytical solution
+#---------------------------------------------------
+
+subsection analytical solution
+  set enable                = true
+  set verbosity             = quiet
+  set filename              = L2Error
+  subsection uvwp
+   set Function expression = 0; 0; if (x * x + y * y < 0.5 * 0.5 , 2, 0)
+  end
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+
+subsection FEM
+  set velocity order = 2
+  set pressure order = 1
+  set VOF order      = 1
```

Comment:
I am curious, what's the influence of the order of the velocity on the quality of the results? I would presume you would still be second-order accurate with degree 1 for velocity no?

Reply:
Yes I think you're right because we are limited by the regularity of the solution. I will try it and update the results if needed :)

Reply:
It changes slightly the results, but not enough to notice it on the figures. I still updated the mesh convergence study figure, but the order stays the same.